core/src/main/java/jenkins/tools/ToolConfigurationCategory.java

```diff
@@ -0,0 +1,21 @@
+package jenkins.tools;
```

Comment:
But `GlobalMavenConfig` is precisely the example of why this should _not_ be restricted: doing so would prevent us from implementing [JENKINS-26462](https://issues.jenkins-ci.org/browse/JENKINS-26462), unless we moved that configuration back into the generic global configuration screen.